src/components/button/button.tsx

```diff
@@ -57,8 +57,8 @@ export const Button = React.forwardRef<any, ButtonProps>(
         color={color}
         variant={variant}
         {...props}
-        to={component === 'link' ? to : null}
-        tabIndex={tabIndex || (isDisabled && component === 'link') ? -1 : null}
+        to={component === 'link' ? to : undefined}
+        tabIndex={tabIndex || (isDisabled && component === 'link') ? -1 : undefined}
```

Comment:
why `undefined`?
cannot find it: https://html.spec.whatwg.org/multipage/interaction.html#attr-tabindex